apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -539,7 +539,7 @@ def create_table_for_html_display(
         for j in range(n):
             each_entry_name = name + str(j + 1)
             raw_submitted_answer = data["raw_submitted_answers"].get(
-                each_entry_name, None
+                each_entry_name, ""
```

Comment:
I couldn't immediately tell which specific change caused this, but in essence, the type of `raw_submitted_answer` is now `Any | None`, which caused an error when we tried to pass it to `escape_unicode_string` (which does not accept `None`).

Reply:
That makes sense 👍🏽